frontend/sozisel-app/src/components/PresenterSession/Agenda/Agenda.tsx

```diff
@@ -0,0 +1,111 @@
+import "./Agenda.scss";
+
+import { Paper, Typography } from "@material-ui/core";
+import React, { ReactElement, useEffect, useState } from "react";
+
+import { AgendaPoint } from "../../../model/Agenda";
+import TurnedInNotIcon from "@material-ui/icons/TurnedInNot";
+
+interface AgendaProps {
+  agendaEntries: AgendaPoint[];
+  estimatedTime: number;
+}
+
+interface AgendaEntryProps {
+  startMinute: number;
+  endMinute: number;
+  name: string;
+  idx: number;
+}
+
+export default function Agenda({
+  agendaEntries,
+  estimatedTime,
+}: AgendaProps): ReactElement {
+  const [counter, setCounter] = useState<number>(0);
+  const agendaStatus = (startMinute: number, endMinute: number) => {
+    if (counter < startMinute) return "";
+    if (
+      (counter >= startMinute && counter < endMinute) ||
+      startMinute === endMinute
+    )
+      return "during";
+    return "finished";
+  };
+
+  useEffect(() => {
```

Comment:
Are you sure that there are no libraries that could do that logic for you? Tried to search for some but found none

Reply:
Yeah, I tried to search but found none which could do this "countdown"